### ISSUE TITLE
[General Mechanic] Glancing Blows incorrect chance and damage reduction

### ISSUE THREAD

**Issue by mercred** [dalaranwow/dalaran-wow#5736]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Currently glancing blows appear 25% of the time and deal 69.5% damage.
**Expected behaviour**: Tell us what should happen instead
Glancing blows should appear 24% of the time and deal 75% damage. https://web.archive.org/web/20100810104916/http://elitistjerks.com/f78/t39136-combat_mutilate_spreadsheets_updated_3_3_a/p125/#post1670144

**Steps to reproduce the problem**:
1. Take low-level weapons with the lowest damage variance
2.  Do 10000 hits on a heroic dummy
3.  Observe recount

**Include proofs for this behaviour**
10k hits against a heroic dummy with 2 same low-level weapons. Average damage of a glancing blow is 69.5% of an average damage of a hit. 
![10k_glancing_recount](https://user-images.githubusercontent.com/15626206/59365507-f1725e00-8d38-11e9-88aa-bbe245d2bac9.jpg)
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Chenocide** [comment]
If you redid the test with no hit rating, you would find that dual wielding miss chance is incorrect too, at about 24.2% +- 0.2%.

**Comment by mercred** [comment]
Bump, seems like an easy fix. There is no incentive to do any additional tests/bug reports when simple bugs are not updated/fixed/replied to for months.
